docs/tutorials/Optics/Optics_ani.md

```diff
@@ -0,0 +1,370 @@
+
+```
+
+and plotting it with xmgrace via
+
+`xmgrace -batch plot_12_together.bat`
+
+This gives us a graph looking like this:
+
+<a id="1_2_together"></a>
+![xx and yy plotted together - identical](1_2_together.png)
+
+There is only 1 line visible - they (almost) perfectly overlap, meaning that $\epsilon_{xx}$ = $\epsilon_{yy}$. Now let's try comparing $\epsilon_{xx}$ with $\epsilon_{zz}$ - all you have to do is make the 2nd block `rut_tens3.dat` (and change the legend appropriately).
+
+![xx and zz plotted together - different](1_3_together.png)
+
+This is a rather interesting result: this seems that the dielectric function in different directions is different - the material is optically anisotropic. Since it is identical in 2 directions, it would be considered birefringent.
+
+## Examining Dielectric Tensor
+
+So now that we have the dielectric function tensor, let's examine how this corresponds to its (anisotropic) optical properties
+<a id = "examining"></a>
+We will run Optados again (no need to rerun Castep), changing the line
+
+```
+OPTICS_GEOM        : tensor    
+```
+
+in the `rut.odi` file to the lines
+
+```
+OPTICS_GEOM        : polarised     # Default
+OPTICS_QDIR : 1 0 0
+```
+
+Let's compare the dielectric function output with the $\epsilon_{xx}$ output we got from the tensor. Run
+
+`xmgrace rut_epsilon.dat tensors/rut_tens1.dat`
+
+We will get the same graph as the [1st one](Optics_ani.md#1_2_together). This is no surprise at all: if light is polarised in the xx direction, it'll effectively feel the $\epsilon_{xx}$ dielectric function.
+
+Let's try a more interesting example - change the direction line to
+
+`OPTICS_QDIR : 1 1 1`
+
+Now the result shouldn't align with anything: to demonstrate that let's compare it to $\epsilon_{xx}$, $\epsilon_{zz}$ and $\epsilon_{xy}$, all on the same graph - luckily xmgrace makes it very easy to plot them together. We'll modify the batch file to include all of them -
+
+*compare.bat*
+```
+
+READ BLOCK "tensors/rut_tens1.dat"
+
+BLOCK xy "1:2"
+S0 LEGEND "Real XX component"
+
+READ BLOCK "tensors/rut_tens3.dat"
+
+BLOCK xy "1:2"
+
+S1 LEGEND "Real ZZ component"
+
+
+READ BLOCK "tensors/rut_tens4.dat"
+
+BLOCK xy "1:2"
+S2 LEGEND "Real XY component"
+
+READ BLOCK "rut_epsilon.dat"
+
+BLOCK xy "1:2"
+
+S3 LEGEND "1 1 1 result"
+
+
+
+XAXIS LABEL "Energy (eV)"
+YAXIS LABEL "Dielectric constant"
+
+WORLD XMAX 5
+```
+
+We have also made the graph smaller (only goes up to 5eV) in the last line to make it easier to see overlaps (or lack of them). This is the kind of output we get:
+
+![Demonstration of 111](111_comparison.png)
+
+You can try comparing it to any other value, but you'll find it's completely different. So how is this related to the tensor?
+
+Considering the properties of tensors in general, we know that the value from a tensor in any direction is given by the equation
+
+$$ {\epsilon}' = \hat{\mathbf{n}}^T \cdot \mathbf{\epsilon} \hat{\mathbf{n}} $$
```

Comment:
Is the first epsilon supposed to be bolded as a vector? Is that why it's in braces?

Reply:
Both epsilons are matrices and n is meant to be a vector - I'm not sure how to distinguish vectors from matrices in LaTex (on paper I just do single underline vs double but I don't think that's standard when doing digitally)

Reply:
There're many conventions:
Vector: `\mathbf` ($\mathbf x$), `\vec` ($\vec x$), `\underline` ($\underline x$)
Matrix: `\mathbf \underline` ($\mathbf{\underline{x}}$), `\underline \underline \mathbf` ($\underline{\underline{\mathbf{x}}}$)

Could also use caps for matrices and lower for vec or any combinations of distinguishing marks you think are best. Making it clear for now means that if a standard is decided upon it can trivially be converteed. A footnote might be useful for this.